app/jobs/scheduled/topics_locale_detection_backfill.rb

```diff
@@ -3,21 +3,31 @@
 module Jobs
   class TopicsLocaleDetectionBackfill < ::Jobs::Scheduled
     every 5.minutes
+    sidekiq_options retry: false
     cluster_concurrency 1
 
     def execute(args)
       return unless SiteSetting.translator_enabled
       return unless SiteSetting.experimental_content_translation
-      return if SiteSetting.automatic_translation_backfill_rate == 0
-
       limit = SiteSetting.automatic_translation_backfill_rate
-      topics =
-        Topic
-          .where(locale: nil)
-          .where(deleted_at: nil)
-          .where("topics.user_id > 0")
-          .order(updated_at: :desc)
-          .limit(limit)
+      return if limit == 0
+
+      topics = Topic.where(locale: nil).where(deleted_at: nil).where("topics.user_id > 0")
```

Comment:
```suggestion
      topics = Topic.where(locale: nil, deleted_at: nil).where("topics.user_id > 0")
```

This is mostly code golfing but there is no need for the overhead of an extra method call even if it may not be signficant.